doc/examples/color_exposure/plot_rgb_to_hsv.py

```diff
@@ -0,0 +1,44 @@
+"""
+================
+RGB to HSV
+================
+
+This example converts an image with RGB channels into an image with HSV (Hue,
+Saturation,Value) channels.
+
+This coordinate system is useful for color-based thresholding since the color
+of a pixel is only described by its Hue value (and not the R,G,B values). [1]_
+
+References
```

Comment:
Remove the section header. See how the references are formatted in other examples (e.g. https://github.com/scikit-image/scikit-image/blob/master/doc/examples/xx_applications/plot_thresholding.py#L24-L29).